x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -9,7 +9,6 @@ import { shallow, ShallowWrapper } from 'enzyme';
 import { EuiButton } from '@elastic/eui';
 
 import { docLinks } from '../../../shared/doc_links';
-
```

Comment:
Can you help me enable [this eslint rule](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/newline-after-import.md)? [Prettier doesn't have an opinion about this](https://prettier.io/docs/en/rationale.html#empty-lines) but if it's something we want to have an opinion about, we should have a rule about it so it can be linted/autofixed rather than deal with this in review.

Reply:
This isn't so much about the newline in and of itself, but more to do with the fact that we reverted changes to this file but missed this one newline, and should revert _all_ changes to the file (to minimize unnecessary git history noise).

I do like the rule you linked and am +1 for adding it - but it wouldn't have caught this specific scenario in any case, because it's not the last import in the list. In this specific scenario my imports are generally grouped by as I've noted in https://github.com/elastic/kibana/pull/89816#discussion_r568993136 (3rd party imports, 1st party shared libs/helpers, 1st party component/view-specific), which is difficult to attempt to automatically lint for. I think you mentioned earlier this is something we should document in our README and I'm definitely down for that.

Reply:
It looks like this still hasn't been addressed.

Reply:
whoops, assumed this was `--fix`ed

Reply:
... it added more deleted lines 😬 

The goal here is to not change/diff `engine_overview_empty.test.tsx` since we're not significantly modifying this file and there's no need for extra git history. Can you add back in the deleted newlines?